### DISCUSSION TITLE
Question about File exchange

### DISCUSSION THREAD

**Discussion by J-Paul0815** [benzino77/tasmocompiler#325] [Q&A]
Is there a way to exchange a (display) driver (same name, different version) before compiling?
Reason see here (at the bottom):
[https://github.com/arendst/Tasmota/discussions/11850](url)

**Comment by benzino77** [comment]
That's what "Custom parameters" field is for. You can put there any `#define`'s or `#undef`'s you want. Have a look at the [description](https://github.com/benzino77/tasmocompiler#how-to-use-custom-parameters)
